src/sdk/pynni/nni/_graph_utils.py

```diff
@@ -8,31 +8,39 @@
 from collections import defaultdict
 import torch
 from torch.utils.tensorboard._pytorch_graph import NodePy, NodePyIO, NodePyOP, GraphPy
-
+from tensorboard.compat.proto.config_pb2 import RunMetadata
+from tensorboard.compat.proto.graph_pb2 import GraphDef
+from tensorboard.compat.proto.step_stats_pb2 import StepStats, DeviceStepStats
+from tensorboard.compat.proto.versions_pb2 import VersionDef
```

Comment:
tensorboard import only used by nas graph, we do not want to import tensorboard if it is not necessary, suggest to move these back.

Reply:
I do agree with that, however, import modules in the function will trigger the pylint warnings which may fail the azure merge check pipeline, and block the merge progress. Is there any way to bypass the pylint check?
`_graph_utils.py:78:8: C0415: Import outside toplevel (tensorboard.compat.proto.config_pb2) (import-outside-toplevel)
_graph_utils.py:79:8: C0415: Import outside toplevel (tensorboard.compat.proto.graph_pb2) (import-outside-toplevel)
_graph_utils.py:80:8: C0415: Import outside toplevel (tensorboard.compat.proto.step_stats_pb2) (import-outside-toplevel)
_graph_utils.py:81:8: C0415: Import outside toplevel (tensorboard.compat.proto.versions_pb2) (import-outside-toplevel)`